c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,10 +80,13 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
-    }));
+    var mapSize = this.map.getSize();
+    if (mapSize) {
+      this.map.getView().fit(app.MapController.DEFAULT_EXTENT, mapSize,
+      /** @type {olx.view.FitOptions} */ ({
```

Comment:
I am wondering if the definition of `olx.view.FitOptions` is not wrong in ol3.

Should `padding` be a required attribute?
https://github.com/openlayers/ol3/blob/master/externs/olx.js#L7406-L7423
http://openlayers.org/en/master/apidoc/ol.View.html#fit